index.ts

```diff
@@ -69,13 +79,13 @@ function initConfig(projectConfig) {
     }
   };
 
-  var envSetConfig = {};
+  let envSetConfig: Config = {};
   if (process.env.hasOwnProperty('GCLOUD_TRACE_CONFIG')) {
-    envSetConfig = require(path.resolve(process.env.GCLOUD_TRACE_CONFIG));
+    envSetConfig = require(path.resolve(process.env.GCLOUD_TRACE_CONFIG)) as Config;
   }
   // Configuration order of precedence:
   // Default < Environment Variable Set Configuration File < Project
-  var config = extend(true, {}, defaultConfig, envSetConfig,
+  const config = extend(true, { forceNewAgent_: false }, defaultConfig, envSetConfig,
```

Comment:
Why was this `forceNewAgent_` necessary? Is it not in `defaultConfig`?

While you're here can you also edit the comment above to add the missing `envConfig` to the precedence list.

Reply:
It's not set in `defaultConfig`. As it's test-only, I don't think we ever want it to be exposed.

Comment edited!